code/src/java/pcgen/core/Equipment.java

```diff
@@ -5510,6 +5470,7 @@ public int getCritMultiplier()
 	 * Gets the altCritMultiplier attribute of the Equipment object
 	 * 
 	 * @return The altCritMultiplier value
+	 * @deprecated due to CRITMULT code control
 	 */
 	public int getAltCritMultiplier()
```

Comment:
the javadoc version does both in Eclipse (and should do so elsewhere), since the javadoc versio predates the ability to annotate code, so no need to do the annotation as well.